src/session/content/message_row/indicators_model.rs

```diff
@@ -88,10 +103,38 @@ impl MessageIndicatorsModel {
 
     pub(crate) fn set_message(&self, message: glib::Object) {
         let imp = self.imp();
-        let old = imp.message.replace(Some(message));
+        let old = imp.message.replace(Some(message.clone()));
```

Comment:
I think it's better to just get the message below the if, so that we can avoid unneeded clones.

Reply:
I now use the `MessageIndicatorsModel::message` method. It also uses a clone, but is much shorter than `self.imp().message.borrow().as_ref().unwrap()`. Feel free to unresolve this if you have a different opinion